templates/zone-master.erb

```diff
@@ -1,6 +1,6 @@
 <%-
-if @is_dynamic and @allow_update.empty?
-  raise(Puppet::ParseError, "allow_update is empty for dynamic zone '#{name}'")
+if @is_dynamic and (@allow_update.empty? and @allow_update_cidr.empty?)
+  raise(Puppet::ParseError, "Both allow_update and allow_update_cidr are empty for dynamic zone '#{name}'")
```

Comment:
Forget it, the change I proposed would break existing configuration in fact.
I'll merge shortly, and we should get a new (working) release shortly (have to add a new feature, for logging support).